src/metabase/models/params/chain_filter.clj

```diff
@@ -193,14 +193,20 @@
   (let [rows (mdb.query/query {:select    [[:fk-field.id :f1]
```

Comment:
Woah this selecting every FK relationship in the database? Surely this is latency + memory issue for large DBs.

Reply:
💯

Reply:
Created a tech debt issue for this https://github.com/metabase/metabase/issues/39843